docker/conf/docker-compose.github-actions.review.simplified.yml.dist

```diff
@@ -34,6 +34,14 @@ services:
             ELASTIC_SEARCH_INDEX_PREFIX: BRANCH_NAME
             REDIS_PREFIX: BRANCH_NAME
             FORCE_ELASTIC_LIMITS: true
+            GOPAY_EN_GOID: GOPAY_GOID
+            GOPAY_EN_CLIENTID: GOPAY_CLIENTID
+            GOPAY_EN_CLIENTSECRET: GOPAY_CLIENTSECRET
+            GOPAY_CS_GOID: GOPAY_GOID
+            GOPAY_CS_CLIENTID: GOPAY_CLIENTID
+            GOPAY_CS_CLIENTSECRET: GOPAY_CLIENTSECRET
+            PACKETERY_API_KEY: PACKETERY_API_KEY_VALUE
```

Comment:
according to the .env file in `project-base/app` it seems like this variable has no effect on the backend